var/spack/repos/builtin/packages/libaec/package.py

```diff
@@ -14,7 +14,7 @@ class Libaec(CMakePackage):
     """
 
     homepage = 'https://gitlab.dkrz.de/k202009/libaec'
-    url = 'https://gitlab.dkrz.de/api/v4/projects/k202009%2Flibaec/repository/archive.tar.gz?sha=v1.0.2'
+    url = 'https://gitlab.dkrz.de/k202009/libaec/-/archive/v1.0.6/libaec-v1.0.6.tar.gz'
```

Comment:
Changes to the URL can affect version checksums which, in this case, it looks like it changes all of the checksums below:

```
    version('1.0.6', sha256='2675d26b24a081cdaaa48e0d085632e568e3f85f843f62b91fc51e1b3f5fe93f')
    version('1.0.5', sha256='563e3f9c614a6c848340eb070f01b4a915e9ebc8a67199254bf37712b92e24d3')
    version('1.0.4', sha256='7d85784afc0c4d82adae2d3a1ad2c0a9db700a49ad8f32674618b0089206cb5e')
    version('1.0.3', sha256='704e5b2d341c32606e21c01217100ec01e7cae05cdef5eeaf4c09ef5adb45583')
    version('1.0.2', sha256='c1aa0d6d8c070f5b1d65a42bc64334fbd69518ea38ab566fb52880bae6f1a72e')
    version('1.0.1', sha256='2495d80375817758a4f1e4b4f6b47a78a7c0d942b4f0899a87045537b0627b56')
    version('1.0.0', sha256='7f25e48a305db75427a3ff426c5f0d489821be5a533c08f3bf503522f674685f')
```